Assets/AnkrSDK/Runtime/WalletConnect2/RpcRequests/SilentSigning/SilentSigningSignMessageRequestData.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+using WalletConnectSharp.Common.Utils;
+using WalletConnectSharp.Network.Models;
+
+namespace AnkrSDK.WalletConnect2.RpcRequests.SilentSigning
+{
+	[RpcMethod("wallet_silentSignMessage")]
+	[RpcRequestOptions(Clock.SIX_HOURS, false, 3005)]
+	[RpcResponseOptions(Clock.SIX_HOURS, false, 3006)]
+	public class SilentSigningSignMessageRequestData : RpcRequestListDataBase
+	{
+		public class SilentSigningSignMessageRequestParams
```

Comment:
same

Reply:
This one is also only used inside enclosing one